### PR TITLE
[New] Change status intent

### PR DESCRIPTION
**Pull request by AdarshNaidu**
This simple intent allows the user to set/change status of his profile

### Conversation
**User**: Change status to I will be busy for the whole of coming week.
**Alexa**: Done
OR
**User**: Change status
**Alexa**: Alright, what's the new status?
**User**: I will be busy for the whole of coming week.
**Alexa**: Done.

### CODE REVIEW

**Comment by TheDreamSaver** [comment]
Are we not using intent/slot confirmation here @AdarshNaidu?

**Comment by AdarshNaidu** [comment]
> Are we not using intent/slot confirmation here @AdarshNaidu?

I thought of keeping the conversation short and concise, since it's just changing status.
Should I keep intent confirmation?

**Comment by TheDreamSaver** [comment]
We should. Because speech to text is never that accurate to get it exactly what you said.
For me, it took something else and set that as a status.

Also, suggest doing all the testing via mic on the console and not the text input until you get a device.

Because the end-user is going to talk and not type.

File: lambda/custom/resources/en-US.json [hunk 1]
```diff
@@ -98,6 +98,7 @@
     },
     "STATUS": {
         "SUCCESS": "Done.",
```
Comment (TheDreamSaver) [inline lambda/custom/resources/en-US.json]: Change it to "Done, I have updated your status."

File: models/en-US.json [hunk 2]
```diff
@@ -478,6 +483,24 @@
...
                     "samples": []
+                },
+                {
+                    "name": "ChangeStatusIntent",
+                    "slots": [
+                        {
+                            "name": "statusmessage",
+                            "type": "AMAZON.SearchQuery",
+                            "samples": [
+                                "{statusmessage}"
```
Comment (TheDreamSaver) [inline models/en-US.json:496]: Try adding 
"change status to {statusmessage}",
"set status to {statusmessage}"

these to the slot elicitation samples as well.

The user once he is asked "Alright, what's the new status?"

He can reply with "set status to {searchquery}"

Currently it is including "set status to" as well.

Check with these two slot utterances if everything works fine,
if yes, add them as well.

**Review by TheDreamSaver — CHANGES_REQUESTED** [review]

**Pull request closed without merging**